docs/filelist.rst

```diff
@@ -24,7 +26,9 @@ You can update metadata or content of these `GoogleDriveFile`_ instances if you
 Paginate and iterate through files
 ----------------------------------
 
-*PyDrive* provides Pythonic way of paginating and iterating through list of files. All you have to do is to limit number of results with ``maxResults`` parameter and build ``for`` loop retrieving file list from API each iteration.
+*PyDrive* provides Pythonic way of paginating and iterating through list of files. 
+All you have to do is to limit number of results with ``maxResults`` parameter and build ``for`` loop retrieving
```

Comment:
Let's change the style a bit, please. Something like this should be better:


Here is an example how to do this, ``maxResults`` below defines how many files it retrieves at once and we wrap it into a `for` loop to iterate: